<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" rowstretch="0" columnstretch="0">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4">
      <item>
       <widget class="QListWidget" name="list_widget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txt_elem_content"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_edit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_find">
            <property name="text">
             <string>find</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="17" column="1">
           <widget class="QPushButton" name="pb_tea">
            <property name="text">
             <string>= чай</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pb_clear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>clear</string>
            </property>
           </widget>
          </item>
          <item row="17" column="2">
           <widget class="QPushButton" name="pb_cakes">
            <property name="text">
             <string>= торты</string>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QPushButton" name="pb_iter_increase">
            <property name="text">
             <string>++</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pb_min_element">
            <property name="text">
             <string>min_element</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QPushButton" name="pb_iter_begin">
            <property name="text">
             <string>begin</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QPushButton" name="pb_inter_decrease">
            <property name="text">
             <string>--</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="pb_max_element">
            <property name="text">
             <string>max_element</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1" colspan="2">
           <widget class="QLabel" name="ldl_workpieces">
            <property name="text">
             <string>Заготовки</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="pb_sort">
            <property name="text">
             <string>merge sort</string>
            </property>
           </widget>
          </item>
          <item row="14" column="2">
           <widget class="QPushButton" name="pb_iter_end">
            <property name="text">
             <string>end</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pb_pop_back">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>pop_back</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLabel" name="lbl_algoriphm">
            <property name="text">
             <string>Алгоритмы</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLabel" name="lbl_method">
            <property name="text">
             <string>Методы</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="pb_shuffle">
            <property name="text">
             <string>shuffle</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pb_unique">
            <property name="text">
             <string>unique</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QPushButton" name="pb_sort_r">
            <property name="text">
             <string>merge sOrT</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pb_erase">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>erase</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pb_insert">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>insert</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pb_push_back">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>push_back</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="lbl_iterator">
            <property name="text">
             <string>Итератор</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="le_count"/>
            </item>
            <item>
             <widget class="QPushButton" name="pb_count">
              <property name="text">
               <string>count</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_count">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="2">
           <widget class="QPushButton" name="pb_reverse">
            <property name="text">
             <string>reverse</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="pb_lower_bound">
            <property name="text">
             <string>lower_bound</string>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QPushButton" name="pb_upper_bourd">
            <property name="text">
             <string>upper_bound</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pb_pop_front">
            <property name="text">
             <string>pop_front</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pb_push_front">
            <property name="text">
             <string>push_front</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_size">
            <property name="text">
             <string>size():</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_size"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_resize">
            <property name="text">
             <string>resize</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
